### ISSUE TITLE
MISSING_MID is inserted if Message-ID is the last header

### ISSUE THREAD

**Issue by patrakov** [vstakhov/rspamd#339]
```
Return-Path: <sender@example.com>
Delivered-To: <victim@example.org>
Received: from example.com (example.com [93.184.216.34])
        by example.org (Postfix) with ESMTPS id 951F43024CE
	for <victim@example.org>; Tue, 21 Jul 2015 09:10:04 +0000 (UTC)
Received: from [192.168.1.2] (unknown [192.0.2.1])
	by example.com (Postfix) with ESMTPS id 62CEB23C666F
	for <victim@example.org>; Tue, 21 Jul 2015 09:10:00 +0000 (UTC)
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: False insertion of MISSING_MID
From: Sender <sender@example.com>
To: Victim <victim@example.org>
Date: Tue, 21 Jul 2015 08:33:13 -0000
Message-ID: <20150721083313.29339.64611@example.com>
X-Dummy-Header: dummy

This (completely fake) message has a message ID!

However, if the dummy header is removed, Rspamd inserts the
MISSING_MID symbol.

Please accept this test message as a bug report.

This report is against this git commit:

$ git show-ref master
def3ec562c8dd789efe2a1f3f31217275fa5ecd3 refs/heads/master
def3ec562c8dd789efe2a1f3f31217275fa5ecd3 refs/remotes/origin/master
```

**Comment by vstakhov** [comment]
That's fixed in master, thanks for report (0.9 seems to be unaffected)!

**Comment by whataboutpereira** [comment]
Hello! I'm experiencing a similar problem with v1.9.4 - this message triggers MISSING_MID.

```
Return-Path: <someone@somewhere.else.com>
Delivered-To: someone@somewhere.com
Received: from localhost
	by somewhere.else.com with LMTP
	id gNquIsnKf11n7gEA+R4aHw
	(envelope-from <someone@somewhere.else.com>)
	for <someone@somewhere.com>; Mon, 16 Sep 2019 20:47:53 +0300
Return-Path: <someone@somewhere.else.com>
Delivered-To: someone@somewhere.com
Received: from localhost (somewhere.else.com [local])
	by somewhere.else.com (OpenSMTPD) with ESMTPA id 7990cd8a
	for <someone@somewhere.com>;
	Mon, 16 Sep 2019 17:47:53 +0000 (UTC)
From: Someone <someone@somewhere.else.com>
Date: Mon, 16 Sep 2019 20:47:53 +0300
To: someone@somewhere.com
Subject: Test
User-Agent: Heirloom mailx 12.5 7/5/10
MIME-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Content-Transfer-Encoding: 7bit
Message-ID: <14e195d4c949fc50@somewhere.else.com>

Test
```
